elastic/kibana
# [ResponseOps][Alerting] Rule run history displays success with a message when the rule status is warning

Resolves https://github.com/elastic/kibana/issues/129120

## Summary

Adds a new field to the event log `kibana.alerting.outcome` that will allow us to set the warning status.


### Checklist


### To verify

_To verify that filtering still works correctly:_
- Run es using `yarn es snapshot --license trial -E path.data=<data_path>` on a different branch
- Create a rule and let it run    
- Run es on this branch with the previous command
- Verify that the rule run history status still shows up and that you can filter on it on the rule details page and in the log tab.

_To verify that warning shows up:_
- Create a rule that will generate a warning status in the rule run history and verify that the status in the table matches the last response 
- I usually generate a 'warning status' by forcing it here `x-pack/plugins/alerting/server/lib/rule_execution_status.ts`
